enterprise/dev/app/build-release.sh

```diff
@@ -22,19 +22,43 @@ bazel_build() {
   cp -vf "${out}" ".bin/sourcegraph-backend-${platform}"
 }
 
+pre_codesign() {
+  # Tauri won't code sign our sidecar sourcegraph-backend Go binary for us, so we need to do it on
+  # our own. https://github.com/tauri-apps/tauri/discussions/2269
+  # For details on code signing, see doc/dev/background-information/app/codesigning.md
+  if [[ ${PLATFORM_IS_MACOS} == 1 ]]; then
+    # We expect the same APPLE_ env vars that Tauri does here, see https://tauri.app/v1/guides/distribution/sign-macos
+    security create-keychain -p my_temporary_keychain_password my_temporary_keychain.keychain
+    security set-keychain-settings my_temporary_keychain.keychain
+    security unlock-keychain -p my_temporary_keychain_password my_temporary_keychain.keychain
+    security list-keychains -d user -s my_temporary_keychain.keychain $(security list-keychains -d user | sed 's/["]//g')
+
+    echo "$APPLE_CERTIFICATE" >cert.p12.base64
+    base64 -d -i cert.p12.base64 -o cert.p12
+
+    security import ./cert.p12 -k my_temporary_keychain.keychain -P "$APPLE_CERTIFICATE_PASSWORD" -T /usr/bin/codesign
+    security set-key-partition-list -S apple-tool:,apple:, -s -k my_temporary_keychain_password -D "$APPLE_SIGNING_IDENTITY" -t private my_temporary_keychain.keychain
+
+    codesign --force -s "$APPLE_SIGNING_IDENTITY" --keychain my_temporary_keychain.keychain --deep .bin/sourcegraph-backend-aarch64-apple-darwin
```

Comment:
I see the macos buiild failed, maybe this needs an absolute path?
```suggestion
    codesign --force -s "$APPLE_SIGNING_IDENTITY" --keychain my_temporary_keychain.keychain --deep "$(pwd)/.bin/sourcegraph-backend-aarch64-apple-darwin"
```